lib/power_api/generator_helper/swagger_helper.rb

```diff
@@ -268,7 +256,7 @@ def spec_tpl_index
         "response '200', '#{resource.plural_titleized} retrieved' do",
           "schema('$ref' => '#/definitions/#{resource.plural}_collection')\n",
           "run_test! do |response|",
-            "expect(JSON.parse(response.body)['data'].count).to eq(expected_collection_count)",
+            "expect(JSON.parse(response.body)['#{resource.plural}'].count).to eq(expected_collection_count)",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [109/100]

Reply:
me gusta como está jojoo